ExtLib/QxOrmSupport.cmake

```diff
@@ -0,0 +1,109 @@
+# -------------------------------------------------------------
+# This function adds the necessary cmake code to find the qxorm
+# shared libraries and setup custom copy commands and/or install
+# rules for Linux and Windows to use
+function(AddQxOrmCopyInstallRules)
+  set(options )
+  set(oneValueArgs )
+  set(multiValueArgs )
+  cmake_parse_arguments(qxorm "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN} )
+  set(INTER_DIR ".")
+
+
+  if(MSVC_IDE)
+    set(qxorm_TYPES Debug Release)
+  else()
+    set(qxorm_TYPES "${CMAKE_BUILD_TYPE}")
+    if("${qxorm_TYPES}" STREQUAL "")
+        set(qxorm_TYPES "Debug")
+    endif()
+  endif()
+
+  if(WIN32)
+    set(qxorm_LIB_TYPE "dll")
+  else()
+    set(qxorm_LIB_TYPE "so")
+  endif()
+
+  set(qxorm_INSTALL_DIR "lib")
+  if(WIN32)
+    set(qxorm_INSTALL_DIR ".")
+  endif()
+
+  set(qxorm_LIB_DIR "${QXORM_DIR}/lib")
+  set(qxorm_LIBVAR "QxOrm")
+  foreach(BTYPE ${qxorm_TYPES} )
+    # message(STATUS "  BTYPE: ${BTYPE}")
+    string(TOUPPER ${BTYPE} UpperBType)
+    if(MSVC_IDE)
+      set(INTER_DIR "${BTYPE}")
+    endif()
+
+    # Set DLL path
+    if(${UpperBType} STREQUAL "DEBUG")
+      set(qxorm_LIBS "QxOrmd")
+    else()
+      set(qxorm_LIBS "QxOrm")
+    endif()
+    set(DllLibPath "${qxorm_LIB_DIR}/${qxorm_LIBS}.${qxorm_LIB_TYPE}")
+
+    # Get the Actual Library Path and create Install and copy rules
+    # get_target_property(DllLibPath ${qxorm_LIBNAME} IMPORTED_LOCATION_${UpperBType})
+    message(STATUS "  DllLibPath: ${DllLibPath}")
```

Comment:
This line should probably be commented out as it is generally used for Debugging